tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -115,13 +115,20 @@ def _check_shared_dir(remote_command_executor, shared_dir):
     ).is_greater_than(0)
 
 
-def _check_auth_ok(remote_command_executor, external_authenticator_port, session_id, session_token):
+def _check_auth_ok(remote_command_executor, external_authenticator_port, session_id, session_token, os):
+    if os == "centos7":
```

Comment:
Instead of a `if/else` can you just do a `string.format()` and substitute username. This makes it more extensible for the future.

Reply:
I'd create an utility function to retrieve the username according to the os and use it also for [remote_command_executor.py](https://github.com/aws/aws-parallelcluster/blob/v2.5.1/tests/integration-tests/remote_command_executor.py#L29)